ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/sflow.j2

```diff
@@ -41,6 +41,14 @@ sFlow is enabled.
 
 sFlow is disabled.
 {%     endif %}
+{%     if sflow.hardware_acceleration.enabled is arista.avd.defined(true) %}
+
+sFlow hardware acceleration is enabled.
```

Comment:
I think we should add some extra info if using modules. Like enabled globally, enabled on modules x, x, x and disabled on modules y, y, y. Or maybe just a table :)

Reply:
Made a doc table! :-)